pkg/worker/k8srunner.go

```diff
@@ -479,3 +579,19 @@ func execInPod(c *rest.Config, namespace, podName string, execOpts *v1.PodExecOp
 func (r *k8sStepRunner) addStatusUpdate(status workermodel.Status) {
 	r.ResultStore.AddStatusUpdate(r.stepID, time.Now(), status)
 }
+
+func validateNoParentDirAccess(p string) bool {
```

Comment:
nit: You're always inverting this function's return value, so I suggest you invert the semantic of it instead.

Such as: `isIllegalParentDirAccess`